advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx

```diff
@@ -0,0 +1,334 @@
+export PGSOCKET="/var/run/edb-pge"
+```
+
+Patroni will bootstrap (create) the initial Postgres cluster and will be in charge of starting the service, so make sure `systemctl` is disabled for this service:
+
+```bash
+$ sudo systemctl disable edb-as-15.service
+```
+or
+
+```bash
+$ sudo systemctl disable edb-pge-15.service
+```
+
+Configure the firewall on each database node so they accept connections to the Postgres service
+
+```bash
+$ sudo firewall-cmd --quiet --zone=public --add-port=$PGPORT/tcp --permanent
+$ sudo firewall-cmd --quiet --reload
+```
+
+## Watchdog
+
+Patroni is the component interacting with the watchdog device. Set the permissions of the software watchdog on both **pg-patroni1** and **pg-patroni2** hosts:
+
+```bash
+$ cat <<EOF | sudo tee /etc/udev/rules.d/99-watchdog.rules
+KERNEL=="watchdog", OWNER="$PGUSER", GROUP="$PGGROUP"
+EOF
+$ sudo sh -c 'echo "softdog" >> /etc/modules-load.d/softdog.conf'
+$ sudo modprobe softdog
+$ sudo chown $PGUSER:$PGGROUP /dev/watchdog
+```
+
+## Patroni
+
+On both **pg-patroni1** and **pg-patroni2** hosts, install Patroni and its dependencies for etcd, following the steps in [Installing Patroni](installing_patroni).
+
+Define the Patroni configuration in `/etc/patroni.yml`:
+
+```bash
+$ CLUSTER_NAME="demo-cluster-1"
+$ MY_NAME=$(hostname --short)
+$ MY_IP=$(hostname -I | awk ' {print $1}')
+$ cat <<EOF | sudo tee /etc/patroni.yml
+scope: $CLUSTER_NAME
+namespace: /db/
+name: $MY_NAME
+
+restapi:
+  listen: "0.0.0.0:8008"
+  connect_address: "$MY_IP:8008"
+  authentication:
+    username: patroni
+    password: mySupeSecretPassword
+
+etcd3:
+    hosts:
+    - etcd1:2379
+    - etcd2:2379
+    - etcd3:2379
+
+bootstrap:
+  dcs:
+    ttl: 30
+    loop_wait: 10
+    retry_timeout: 10
+    maximum_lag_on_failover: 1048576
+    postgresql:
+      use_pg_rewind: true
+      use_slots: true
+      parameters:
+        archive_mode: "on"
+        archive_command: "/bin/true"
+
+  initdb:
+  - encoding: UTF8
+  - data-checksums
+  - auth-local: peer
+  - auth-host: scram-sha-256
+
+  pg_hba:
+  - host replication replicator 0.0.0.0/0 scram-sha-256
+  - host all all 0.0.0.0/0 scram-sha-256
+
+  # Some additional users which needs to be created after initializing new cluster
+  users:
+    admin:
+      password: admin%
+      options:
+        - createrole
+        - createdb
+
+postgresql:
+  listen: "0.0.0.0:$PGPORT"
+  connect_address: "$MY_IP:$PGPORT"
+  data_dir: $PGDATA
+  bin_dir: $PGBIN
+  pgpass: /tmp/pgpass0
+  authentication:
+    replication:
+      username: replicator
+      password: confidential
+    superuser:
+      username: $PGUSER
+      password: my-super-password
+    rewind:
+      username: rewind_user
+      password: rewind_password
+  parameters:
+    unix_socket_directories: "$PGSOCKET,/tmp"
+
+watchdog:
+  mode: required
+  device: /dev/watchdog
+  safety_margin: 5
+
+tags:
+  nofailover: false
+  noloadbalance: false
+  clonefrom: false
+  nosync: false
+EOF
+```
+
```

Comment:
Maybe add a few lines explaining the `replication` and `rewind` authentication section. Or the whole authentication section so the customer knows that those users with the passwords specified need to be present in the cluster